pkg/apis/veleroplugin/v1/upload.go

```diff
@@ -35,6 +35,11 @@ type UploadSpec struct {
 	// BackupRepository provides backup repository info for upload. Used for
 	// multiple backup repository.
 	BackupRepositoryName string `json:"backupRepository,omitempty"`
+
+	// SanpshotReference is the namespace and snapshot name for this upload request.
```

Comment:
SanpshotReference -> SnapshotReference
Please give an example of SnapshotReference in the comment.  Is it namespace/<snapshot name> or some other format?

Reply:
SnapshotNamespace:SnapshotCRName